homeassistant/components/sensor/mqtt.py

```diff
@@ -104,6 +111,20 @@ def message_received(topic, payload, qos):
                 self._expiration_trigger = async_track_point_in_utc_time(
                     self.hass, self.value_is_expired, expiration_at)
 
+            if self._json_attributes:
```

Comment:
Nitpick: I personally strongly prefer `if len(self._json_attributes) > 0`. I'm always getting confused and assume things that are being used as boolean expression to be booleans.

Reply:
Done, will upload in next commit

Reply:
No, the pythonic way is to do `if self._json_attributes` to check if the container is non empty. It's also the preferred way for home assistant.